access-ocp-api-through-cis.md

```diff
@@ -0,0 +1,104 @@
+## Accessing the api endpoint deployed on OpenShift through CIS
+
+1. Setup an OpenShift cluster with ingress. The cluster comes with a default endpoint `/healthz` that is accessible as:
+   ```
+   % oc get routes -n openshift-ingress
+   NAME                                      HOST/PORT                                                                               PATH       SERVICES                  PORT   TERMINATION     WILDCARD
+   route.route.openshift.io/router-default   router-default.xxx-3b5bf5f75xxxx21c8c35ad277-0000.us-south.containers.appdomain.cloud   /healthz   router-internal-default   1936   edge/Redirect   None
+
+   % curl https://router-default.xxx-3b5bf5f75xxxx21c8c35ad277-0000.us-south.containers.appdomain.cloud/healthz
+   ok
+   ```
+
+   Alternatively you deploy your app and create the NodePort service for the application endpoint.
+
+2. Open the CIS service and navigate to `DNS` tab under `Reliability`.
+
+3. Go to `DNS records` and add a new record:
+   ```
+   Type: CNAME
+   Name: <any name>
+   TTL: Automatic
+   Alias domain name: <openshift_route>  ## example: router-default.xxx-3b5bf5f75xxxx21c8c35ad277-0000.us-south.containers.appdomain.cloud
+   ```
+
+4. If CIS domain is `example.com` and DNS record name provided in step 3 is `test` then the endpoint can be accessed via `https://test.example.com/<api_endpoint>`. If you access the link now, you will get `ssl handshake failure` error as below.
```

Comment:
This is not a step, but an explanation. So remove the number and indent under `3`. 

(Tip, in markdown, start all your ordered list items with `1. `. Then when you move or remove steps, you don't have to renumber. Apologies if you knew that already).

Reply:
Made the required changes.